int8 DO_Pin0
int8 DO_Pin1
char DO_Val0
char DO_Val1